libkbfs/bserver_disk.go

```diff
@@ -0,0 +1,240 @@
+// Copyright 2016 Keybase Inc. All rights reserved.
+// Use of this source code is governed by a BSD
+// license that can be found in the LICENSE file.
+
+package libkbfs
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"sync"
+
+	"github.com/keybase/client/go/logger"
+	"golang.org/x/net/context"
+)
+
+// BlockServerDisk implements the BlockServer interface by just
+// storing blocks in a local leveldb instance
```

Comment:
Several more below.  (I don't actually care much, but I think I've seen you point it out before so I thought you might want to know.)